DOM_creation/Lesson2/main.js

```diff
@@ -0,0 +1,15 @@
+const ul= document.getElementById("js-ul");
+const li = document.createElement('li');
+
+const a = document.createElement('a');
+a.setAttribute("href", "1.html");
+const text = document.createTextNode("これです");
```

Comment:
Q Do you have a reason you used createTextNode here?

Reply:
It might be an overwrap with the comet above, the reason why I used createTextNode is because I wanted to avoid <img> tag to have </img> closure tag.

First I used code below to append text to img tag,
img.textContent = 'これです’;
But when it was rendered into HTML, it had a <img> closure tag and the text was not displayed. 

I could not figure out other ways to insert the text between <img> tag and </a> tag.

Reply:
![Screen Shot 2022-09-06 at 7 34 11 pm](https://user-images.githubusercontent.com/86552484/188600878-e000e385-bd4b-47c8-8a23-517eeced16fc.png)

Reply:
![Screen Shot 2022-09-06 at 7 34 44 pm](https://user-images.githubusercontent.com/86552484/188601073-0ce438c7-0324-4780-a1c1-a9c2700a9b85.png)

Reply:
Please see the screenshots. Hope it makes sense
Please let me know if you have questions.

Reply:
Please disregard the comments above,
I somehow found the method which appends img tag right after a start tag. 

Using insertAdjacentElement and selecting position as 'afterbegin' fixed the problem!